src/pages/Popup/index.tsx

```diff
@@ -9,27 +6,14 @@ import './index.css';
 
 declare const VERSION: string;
 
-async function restoreSession() {
-  const { pHash } = await chrome.storage.session.get('pHash');
-  const { walletState } = rootStore.store.getState();
-  if (pHash) {
-    const seed = await decryptMnemonic(walletState.encryptedSeed, pHash);
-    rootStore.store.dispatch(setWalletSeedPhraseAction(seed));
-  }
-}
-
 const renderApp = async () => {
-  await restoreSession();
   persistQueryClient({
     queryClient,
     persister: offlineStorage,
     buster: VERSION,
   });
   const container = document.getElementById('app');
 
-  // We need to do this after load in order for the extension popup to have the correct height on first load
-  container!.style.maxHeight = '100vh';
-
```

Comment:
@victorkirov do we not need this anymore?

Reply:
Nope, this was replaced by another mechanism. See the useEffect here:
https://github.com/secretkeylabs/xverse-web-extension/blob/f0b289c725a8b447fc6918b4c2ac62537658a664/src/app/components/screenContainer/index.tsx